content/release-notes/1.20.1.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2020-10-14
+linktitle: "1.20.1"
+title: "1.20.1"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<fixes>}}
+* Fixed a bug that caused getting the command to generate a support bundle to fail in the Admin Console.
```

Comment:
@salahalsaleh this sentence is awkward. What exactly is failing? is it "getting the command" or the command itself that fails?

Reply:
getting the command is what's failing